mrpt_graphslam_2d/include/mrpt_bridge/vrep_conversion.h

```diff
@@ -0,0 +1,36 @@
+#ifndef VREP_CONVERSION_H
+#define VREP_CONVERSION_H
+
+#include <stdint.h>
+#include <string>
+
+#include <mrpt/version.h>
+
+#include "extApi.h"
+
+namespace mrpt
+{
+namespace obs
+{
```

Comment:
Given the  [C++17 PR](https://github.com/shubham-kumar1410/mrpt_vrep_bridge/pull/1) you can now use [nested namespaces](http://www.nuonsoft.com/blog/2017/08/01/c17-nested-namespaces/) to cleanup such parts of code